### PR TITLE
Dludwig/css grid

### PR DESCRIPTION
**Pull request by dsludwig**
Another attempt at #2150, using CSS grid to layout, and without Bootstrap or other external libraries.

### CODE REVIEW

**Comment by mrocklin** [comment]
Hrm, I'm getting this on my machine

image

If I resize the window then things become quite large.  This looks like the resizing issue we've seen on the graph and tasks pages as well.

**Comment by mrocklin** [comment]
I suspect that this is due to https://github.com/bokeh/bokeh/issues/8115

**Comment by mrocklin** [comment]
Do things work for you @dsludwig ?  I'm surprised that there is a difference between our two environments.

**Comment by dsludwig** [comment]
I must have broken something while rolling back the Bootstrap part. It should work now.

**Comment by mrocklin** [comment]
cc @birdsarah this work may interest you.

File: distributed/bokeh/templates/simple.html [hunk 1]
```diff
@@ -1,8 +1,6 @@
 {% extends "base.html" %}
 
 {% block content %}
-<div class="dashboard">
-    {{ plot_div }}
-</div>
+{{ plot_div }}
```
Comment (mrocklin) [inline distributed/bokeh/templates/simple.html]: @birdsarah this fixes the bokeh small-graph issue.  I don't know why

**Comment by mrocklin** [comment]
From my perpsective this PR is pretty much ready to go in.   Additional nice-to-haves would be to arrange the system and profile pages as well.

-  **System:** In widescreen mode it would be good to arrange the plots in a 2x2 square grid, in narrow-mode it would be good to have them in a 4x1 grid with four rows and one column
-  **Profile:** I'm not sure what is best here.  I think that we probably want a maximum width on the content for this page so that it doesn't blow up too much in widescreen mode.

**Comment by birdsarah** [comment]
I'm so excited to see this. When will it be released?

**Comment by mrocklin** [comment]
> I'm so excited to see this. When will it be released?

Possibly quite soon.  I'm planning to release a couple dask projects this week.  I hope that this gets in in time.  Given the potentially fast turnaround on the release it would be very helpful if people can give this a try and report any usability issues.  This would help to allay concerns about releasing fresh code without having it simmer in master for a while.

**Comment by birdsarah** [comment]
Cool. I expect to be using dask again tomorrow so will check it out.

**Comment by mrocklin** [comment]
I'm going to go ahead and merge this in for now.  I think that it is a strict improvement.  

Thank you @canavandl and @dsludwig for your work on this.  You've made me quite happy :)

**Comment by birdsarah** [comment]
quite is the highest on @mrocklin's happiness scale :D

**Pull request merged**